crates/sdk/src/keygen/static_verifier.rs

```diff
@@ -1,21 +1,27 @@
+#[cfg(any(feature = "evm-prove", feature = "evm-verify"))]
```

Comment:
remove all of these cfg and just feature gate the entire `static_verifier` module